modules/ROOT/pages/viewing-problems-fd-canvas.adoc

```diff
@@ -0,0 +1,24 @@
+= List and Navigate to Errors that are in a Mule-App Project
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+:imagesdir: ../assets/images
+
+You can get help understanding the errors in a project and finding out where to fix the errors. The *Problems* pane lists and describes errors, and you can click an error to be taken to the location of the error in a project.
```

Comment:
@rddheath it might be helpful to explain that we support/show 4 different types of Problems in Flow Designer: 
1. Connector Card Errors https://mulesoft.slack.com/files/UD91XTWAD/FGVCCF3MK/card_error.gif
2. Flow Errors https://mulesoft.slack.com/files/UD91XTWAD/FGUA13VB2/flow_error.gif 
3. Connector Config Error https://mulesoft.slack.com/files/UD91XTWAD/FGV2PJF1Q/config_error.gif
4. Project Errors

Reply:
Fixed. See https://github.com/mulesoft/docs-design-center/blob/DOCS-4339-FD-1-9-0-rh/modules/ROOT/pages/viewing-problems-fd-canvas.adoc.